services/apps/alcs/src/portal/public/search/public-search.controller.ts

```diff
@@ -247,6 +250,7 @@ export class PublicSearchController {
       ownerName: notification.applicant,
       class: 'NOTI',
       status: notification.status.status_type_code,
+      outcome: null
```

Comment:
Since SRW's don't have outcomes, can you make the field optional so we don't need to set null here?